lib/ecto/adapters/postgres/connection.ex

```diff
@@ -702,6 +719,19 @@ if Code.ensure_loaded?(Postgrex.Connection) do
     defp null_expr(true), do: "NULL"
     defp null_expr(_), do: []
 
+    defp new_constraint_expr(%Ecto.Migration.Constraint{check: nil, exclude: nil})  do
```

Comment:
Actually, I just realized you did those checks on the runner, so you can get rid of the first two clauses here and the nil checks below.